google/cloud/spanner/ci/run_integration_tests_emulator_cmake.sh

```diff
@@ -0,0 +1,126 @@
+#!/usr/bin/env bash
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -eu
+
+if [[ $# -lt 1 ]]; then
+  echo "Usage: $(basename "$0") <binary-dir> [ctest-args]"
+  exit 1
+fi
+
+BINARY_DIR="$(
+  cd "${1}"
+  pwd
+)"
+readonly BINARY_DIR
+shift
+ctest_args=("$@")
+
+CMDDIR="$(dirname "$0")"
```

Comment:
If you source init.sh, this would be `$PROGRAM_DIR`

Reply:
This is no longer applicable I thinkg.